src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.model;
+
+public enum Operation {
```

Comment:
It's not a model and is used only in fruitTransaction so can be moved to  fruitTransaction inner enum